src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -12,7 +12,12 @@
 import java.security.Principal;
 import java.time.ZonedDateTime;
 import java.time.temporal.ChronoUnit;
-import java.util.*;
```

Comment:
These should be compressed back to star imports. Afaik we use 4 as our threshold for this

Reply:
My Intellij is behaving so weird.. Guess autoformatters are fighting against each other